Specs/Scene/SceneTransformsSpec.js

```diff
@@ -192,4 +192,19 @@ defineSuite([
         expect(drawingBufferCoordinates.x).toBeLessThan(1.0);
         expect(drawingBufferCoordinates.y).toBeLessThan(1.0);
     });
+
+    it('should not error when zoomed out and in 2D', function(done) {
+        var scene = createScene();
```

Comment:
I had to create a new scene instead of using the one that was created in the beforeAll block because changes I made to it were impacting other tests. I can update this spec be a little less fragile by creating the scene in the beforeEach instead of just once if that's preferred.